src/nextjs/templates.ts

```diff
@@ -151,7 +160,9 @@ Sentry.init({
   tracesSampleRate: 1,
 
   // Setting this option to true will print useful information to the console while you're setting up Sentry.
-  debug: false,${additionalOptions}
+  debug: false,
+  ${additionalOptions}
+  ${spotlightOption}
```

Comment:
Looking at this change superficially I think this will mess up the output.

Reply:
Hi @lforst this was adding some extra empty lines, which I just fixed.

Is there any specific issue that you are mentioning here? which I am missing?

Reply:
That was what I was referring to. Thanks for fixing it!